lib/iris/fileformats/nimrod_load_rules.py

```diff
@@ -25,58 +28,306 @@
 NIMROD_DEFAULT = -32767.0
 
 TIME_UNIT = cf_units.Unit(
-    "hours since 1970-01-01 00:00:00", calendar=cf_units.CALENDAR_GREGORIAN
+    "seconds since 1970-01-01 00:00:00", calendar=cf_units.CALENDAR_GREGORIAN
 )
 
 
+DEFAULT_UNITS = {
```

Comment:
This definition is currently only used by 'probability_coord'.
Unless it has possibly broader future scope, I'd be tempted to move it there, and even embed it in the routine?

Reply:
Moved and renamed to make it clearer what it's purpose is.